src/app/components/form/form.component.html

```diff
@@ -53,182 +59,232 @@ <h1 i18n>Domain name</h1>
+                  <div id="digest-{{i}}-error-required" i18n class="invalid-feedback" *ngIf="ds_info.controls.digest.errors?.required" >
+                    Digest required
+                  </div>
                 </div>
-              </div>
 
-              <div class="col-auto">
-                <button (click)="deleteRow('nameservers', i)"
-                class="btn btn-danger delete">
-                  <i class="fa fa-trash" aria-hidden="true"></i>
-                </button>
-              </div>
+                <div class="col-auto">
+                  <button
+                          (click)="deleteRow('ds_info',i)"
+                          class="btn btn-danger delete"
+                          i18n-title
+                          title="Delete DS record">
+                    <i class="fa fa-trash" aria-hidden="true"></i>
+                  </button>
+                </div>
 
-              <div class="col-auto">
-                <button *ngIf="i == form.controls.nameservers.length - 1"
-                        type="button"
-                        (click)="addNewRow('nameservers')"
-                        class="btn btn-success add">
-                  <i class="fa fa-plus" aria-hidden="true"></i>
-                </button>
+                <div class="col-auto">
+                  <button *ngIf="i == form.controls.ds_info.controls.length - 1"
+                          type="button"
+                          (click)="addNewRow('ds_info')"
+                          class="btn btn-success add"
+                          i18n-title
+                          title="Add new DS record">
+                    <i class="fa fa-plus" aria-hidden="true"></i>
+                  </button>
+                </div>
               </div>
             </div>
-          </div>
+            <div class="col-auto">
+              <button type="button" class="btn btn-secondary fetchDataFromParent" (click)="fetchDataFromParent('DS')" i18n>Fetch DS from parent zone</button>
+            </div>
+          </fieldset>
 
           <hr>
 
-          <h4 i18n>Delegation Signers (DS records)</h4>
-          <div formArrayName="ds_info">
-            <div class="row mb-2" *ngFor="let ds_info of form.controls.ds_info.controls; let i=index" [formGroupName]="i">
-              <div class="col-md-2">
-                <input formControlName="keytag" class="form-control" i18n-placeholder placeholder="Key Tag"
-                       [class.is-invalid]="ds_info.controls.keytag.invalid && (ds_info.controls.keytag.dirty || ds_info.controls.keytag.touched)">
-                <div class="invalid-feedback" *ngIf="ds_info.controls.keytag.errors?.serverError" >
-                  {{ ds_info.controls.keytag.errors.serverError }}
+          <fieldset>
+            <legend i18n>General</legend>
+            <div class="row">
+              <fieldset aria-describedby="protocol-error" class="col-md-5 form-col">
+                <legend for="profile">Protocol</legend>
```

Comment:
"Protocol" does not add anything. It is better to remove.

Reply:
I am not sure about that. It adds a grouping label for the two check box and add visual balance when the profile select is present.

If I remove it then I if don't do any think the checkbox are not aligned with the select any more:
![Screenshot 2022-11-24 at 11-27-23 Zonemaster](https://user-images.githubusercontent.com/19394895/203761112-e9eb1f4d-e37b-4025-8f6c-4fb97cad25a7.png)

If I add some margin to the checkbox it can work when the select is present
![Screenshot 2022-11-24 at 11-27-11 Zonemaster](https://user-images.githubusercontent.com/19394895/203761215-07b61530-8e7b-4268-8ee6-c5112ab8b106.png)

but introduced a big gap when the select is hidden

![Screenshot 2022-11-24 at 11-26-51 Zonemaster](https://user-images.githubusercontent.com/19394895/203761267-562a2a4e-e195-4d6e-858b-6d9e99f04b8a.png)

Reply:
I think this is useful to have as well.

Reply:
> I think this is useful to have as well.

For whom do you think it adds anything?

Reply:
> I am not sure about that. It adds a grouping label for the two check boxes and add visual balance when the profile select is present.

Would it work with 
```
Profile: [default                      ^]
```
aligned with the check boxes?

Reply:
If the alignement issue can be resolved, this is fine for me to remove the title "Protocol"

Reply:
I will test that.